### PR TITLE
Remove styled components from TSNE page

### PR DESCRIPTION
**Pull request by clari182**
- Helps with #1617

### CODE REVIEW

**Comment by clari182** [comment]
Sorry @lmcnulty , not ready for review yet! My bad

**Comment by clari182** [comment]
Ready for review!

File: site/gatsby-site/src/tailwind.css [hunk 1]
```diff
@@ -558,4 +558,40 @@
...
+    .tsne-visualization-wrapper>.react-transform-wrapper>.react-transform-component {
+        @apply h-[calc(100vh-2rem)] w-[calc(100vh-2rem)] overflow-visible !important;
+    }
+
+    .tsne-visualization-item>a:not(.incident-card) {
+        @apply text-inherit absolute text-[75%] h-[2em] w-[2em] px-[2px] rounded-[50%] -ml-4 -mt-4 z-2 text-center leading-8 shadow-tsne flex justify-center items-center;
+    }
+
+    .tsne-visualization-item .incident-card {
+        @apply block min-h-[15em] w-60 rounded-[0.25rem] shadow-tsne absolute p-4 overflow-hidden;
```
Comment (lmcnulty) [inline site/gatsby-site/src/tailwind.css:574]: image

I think this also needs `bg-white`.

**Review by lmcnulty — CHANGES_REQUESTED** [review]
image

When you color in the white background, the bottom of the card image preview is rounded when it should be flat.

**Comment by lmcnulty** [comment]
Thanks! Some more things:

When zoomed in heavily, the card offset is weird. Also, the plot point should not turn white when hovered.

<table>
<tr>
<th>This PR</th>
<th>Current</th>
</tr>
<tr>
<td></td> 
<td></td>
</tr>
</table>

The plot point opacity should have a transition:

<table>
<tr>
<th>This PR</th>
<th>Current</th>
</tr>
<tr>
<td><video src="https://user-images.githubusercontent.com/25443411/227299637-7b51dc77-28f0-4539-96fe-2891389edcf8.mp4"/></td>

<td><video src="https://user-images.githubusercontent.com/25443411/227298755-9fce6107-dd93-4308-b15f-e5cda15d82d1.mp4"/></td>
</tr>
</table>

The points with classifications should appear above the unclassified ones:

<table>
<tr>
<th>This PR</th>
<th>Current</th>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

**Comment by clari182** [comment]
Ready for another round of review @lmcnulty !

**Review by lmcnulty — APPROVED** [review]

**Comment by cesarvarela** [comment]
@clari182 please fix conflicts

**Comment by clari182** [comment]
Done @cesarvarela !

**Pull request merged**